iree/compiler/Dialect/Flow/Transforms/DispatchLinalgOnTensors.cpp

```diff
@@ -1017,6 +1017,27 @@ static unsigned makeElementwiseOpsRootOps(FuncOp funcOp, unsigned numRoots) {
   return numRoots;
 }
 
+/// Sets fill op as root op if they are not part of any fusion groups yet.
+static unsigned makeFillOpsRootOps(FuncOp funcOp, unsigned numRoots) {
+  MLIRContext *context = funcOp.getContext();
+  OpBuilder builder(context);
+  for (Block &block : funcOp) {
+    auto linalgOps = block.getOps<linalg::LinalgOp>();
+    for (linalg::LinalgOp linalgOp : llvm::reverse(linalgOps)) {
```

Comment:
Nit: Doesnt need to be `reverse`

Reply:
I guess the order doesn't matter, let me simplify that.

Reply:
simplified this function a bit